cmd/dra-example-controller/claimparametersgen.go

```diff
@@ -0,0 +1,229 @@
+package main
```

Comment:
It would be good to have a copiright header is all .go files.

Reply:
Added